torch/testing/_internal/common_methods_invocations.py

```diff
@@ -143,6 +143,45 @@ def close_to_int(x, eps=0.1):
     return (y < eps) | (y > (1 - eps))
 
 
+def sample_inputs_slice(op_info, device, dtype, requires_grad, **kwargs):
+    for i in [
+        SampleInput(
+            input=torch.ones(3, device=device, dtype=dtype, requires_grad=requires_grad),
+            args=(0,),
+        ),
+        SampleInput(
+            input=torch.randn((20, 30, 40), device=device, dtype=dtype, requires_grad=requires_grad),
```

Comment:
Instead of directly calling randn multiple times this should partial `make_tensor` for a more succinct and less error-prone call